assets/app/view/game/round/stock.rb

```diff
@@ -68,7 +70,7 @@ def render
           children << h(SpecialBuy) if @current_actions.include?('special_buy')
           children.concat(render_failed_merge) if @current_actions.include?('failed_merge')
           children.concat(render_bank_companies) if @bank_first
-          children.concat(render_corporations)
+          children.concat(render_corporations) unless @hide_corporations
```

Comment:
what's this for?

Reply:
When claiming the award for private woosong (implemented as sell_company action), I need to hide the corporations during a stock round, so that only that private is visible, and it's clear what the player needs to do (claim or pass).
Otherwise the private shows up after the corporations, and since there are 16 of them it's really not clear what the user needs to do.

Reply:
this is what it looks like with hidden corps:
<img width="569" alt="image" src="https://user-images.githubusercontent.com/108803147/214010791-04c90354-7753-46ae-96cc-5b583fcdd619.png">

Reply:
It isn't clear to me from the UI what the player is supposed to do. Would this be clearer as a choice action? The 1822 cash doubler works like the latter (and happens at the end of the SR). To me, its clearer. What do you think?

https://18xx.games/game/110760?action=144

Reply:
<img width="450" alt="image" src="https://user-images.githubusercontent.com/108803147/214233769-62c4fbda-1776-41e2-aa36-addce6320276.png">

Reply:
new ui, now that it's a choose and not a sell_company action